app/helpers/common_parameters_helper.rb

```diff
@@ -17,4 +17,8 @@ def parameter_value_field(value)
       content_tag(:span, :class => "help-block") { popover(_("Additional info"), _("<b>Source:</b> %{type} %{name}") % {:type => _(value[:source].to_s), :name => source_name})}
     end
   end
+
+  def use_puppet_default_help link_title = _("Explain use Puppet default"), title = _("Use Puppet Default")
+    popover(link_title, _("Do not send this parameter via the ENC.<br> Puppet will use the value defined in the puppet manifest for this parameter"), :title => title)
```

Comment:
stray space after the break, I suggest you drop the second "puppet" and just say "manifest", add an ending full stop.